src/i18n/languages.ts

```diff
@@ -4,7 +4,7 @@ export const APP_LOCALES = {
   'zh-HANT': '中文 (Chinese Traditional)',
   cs: 'Čeština',
   da: 'Dansk',
-  nl: 'Nederlands',
+  nl: 'Dutch',
```

Comment:
Why change? All other languages in this list are in the native language as well

Reply:
Idk, i just followed the @nathanaelhoun comment on the issue. Nevertheless, crowdin maps Netherlands as Dutch I think

I'm fine on not having that change 🙈

Reply:
Actually, @cino is right, I didn't understood why but it's because Nederlands is the native name. Can you revert this one @SpecialAro?

Reply:
As it was my error in the first place, I did it in `b45a0ac3`